conversations/v3/common.yaml

```
@@ -361,7 +361,7 @@ components:
         to:
           type: string
           description: |
-            The receipient of the messages
+            The recipient of the messages
```

```suggestion
            The message's recipient
```
The baseline is sending one message. So, I suggest using singular and not plural (not counting bulk messaging).